src/test/fuzz/fuzz.cpp

```diff
@@ -72,7 +72,11 @@ extern "C" int LLVMFuzzerInitialize(int* argc, char*** argv)
 }
 
 #if defined(PROVIDE_MAIN_FUNCTION)
+#ifdef WIN32
+int main(int args, char **argv)
+#else
 __attribute__((weak)) int main(int argc, char** argv)
+#endif
```

Comment:
I don't really like changing code in a build change. Maybe exclude the failing ci config via `--disable-fuzz` for now?